corehq/apps/api/resources/messaging_event/serializers.py

```diff
@@ -104,23 +104,18 @@ def _serialize_event_form(event):
 def _get_messages_for_email(event):
     try:
         email = Email.objects.get(messaging_subevent=event.pk)
-        content = email.body
-        recipient_address = email.recipient_address
-        message_id = email.id
     except Email.DoesNotExist:
-        content = None
-        recipient_address = None
-        message_id = None
+        return []
```

Comment:
this change isn't directly related to this PR but it brings email's in line with sms messages (empty messages if none are found)